apps/web/src/pages/brand/tabs/LayoutsListPage.tsx

```diff
@@ -0,0 +1,156 @@
+import { ColumnWithStrictAccessor } from 'react-table';
+import { Data, Table } from '../../../design-system/table/Table';
+import { Button, colors, Tag, Text, Tooltip } from '../../../design-system';
+import { format } from 'date-fns';
+import { ActionIcon, useMantineTheme } from '@mantine/core';
+import { Edit, PlusCircle, PlusGradient, Trash } from '../../../design-system/icons';
+import styled from '@emotion/styled';
+import { useState } from 'react';
+import { LayoutEditor } from './LayoutEditor';
+import { DeleteConfirmModal } from '../../../components/templates/DeleteConfirmModal';
+import { When } from '../../../components/utils/When';
+
+const data = [
+  { id: '1', name: 'Try 1', createdAt: new Date(), updatedAt: new Date(), used: true },
+  { id: '2', name: 'Try 2', createdAt: new Date(), updatedAt: new Date(), used: false },
+];
+
+const enum ActivePageEnum {
+  LAYOUTS_LIST = 'layouts_list',
+  EDIT_LAYOUT = 'edit_layout',
+  CREATE_LAYOUT = 'create_layout',
+}
+
+export function LayoutsListPage() {
+  const theme = useMantineTheme();
+  const [layouts, setLayouts] = useState(data);
+  const [editId, setEditId] = useState('');
+  const [activeScreen, setActiveScreen] = useState(ActivePageEnum.LAYOUTS_LIST);
+  const [toDelete, setToDelete] = useState('');
+
+  const cancelDelete = () => {
+    setToDelete('');
+  };
+
+  const onDelete = (id) => {
+    setToDelete(id);
+  };
+
+  const confirmDelete = () => {
+    handleDeleteLayout(toDelete);
+    setToDelete('');
+  };
+
+  const handleDeleteLayout = (layoutId: string) => {};
+
+  const goBack = () => {
+    setActiveScreen(ActivePageEnum.LAYOUTS_LIST);
+    setEditId('');
+  };
+
+  const editLayout = (id: string) => {
+    setEditId(id);
+    setActiveScreen(ActivePageEnum.EDIT_LAYOUT);
+  };
+
+  const columns: ColumnWithStrictAccessor<Data>[] = [
+    {
+      accessor: 'name',
+      Header: 'Name',
+      Cell: ({ name }: any) => (
+        <Tooltip label={name}>
+          <Text rows={1}>{name}</Text>
+        </Tooltip>
+      ),
+    },
+    {
+      accessor: 'createdAt',
+      Header: 'Created At',
+      Cell: ({ createdAt }: any) => format(new Date(createdAt), 'dd/MM/yyyy HH:mm'),
+    },
+    {
+      accessor: 'updatedAt',
+      Header: 'Last Updated',
+      Cell: ({ updatedAt }: any) => format(new Date(updatedAt), 'dd/MM/yyyy HH:mm'),
```

Comment:
We can have a function inside of this file that abstracts the format functionality like:
```
const formatDate = (date: string): string => format(new Date(date), 'dd/MM/yyyy HH:mm');
```
So then we can use it like:
```
      Cell: ({ createdAt }: any) => formatDate(createdAt),
...
      Cell: ({ updatedAt }: any) => formatDate(updatedAt),
```

Reply:
Sure